rfcs/0112-demote-darwin-support-tier.md

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
I wonder if #88 could help, by leading to a workflow «these things cannot be fixed quickly, so we go forward on Linux — then we pin old versions for Darwin»

Reply:
(By the way, #88 has been accepted and merged in the meantime)